skxray/cdi.py

```diff
@@ -109,7 +115,7 @@ def convolution(array1, array2):
 
     Returns
     -------
-    array :
+    arr : array
```

Comment:
This function needs to be renamed to something like 'approximate_convolution' and have a `.. warning` which says something to the effect of "This function is incorrect but runs faster; use at your own risk".

It also needs a warning that it only works in the case where the two inputs are the same length (which is very surprising).